webpack.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  context: __dirname + "/src",
+  entry: './index.ts',
+  output: {
+    path: __dirname + '/browser',
+    filename: 'bundle.js',
+    library: 'subscriptions_transport_ws'
```

Comment:
Usually exported libs are in pascal case: `'SubscriptionsTransportWs'`, please do that!